modules/liveIntentIdSystem.js

```diff
@@ -7,7 +7,7 @@
 import { triggerPixel, logError } from '../src/utils.js';
 import { ajaxBuilder } from '../src/ajax.js';
 import { submodule } from '../src/hook.js';
-import { LiveConnect } from 'live-connect-js/esm/initializer.js';
+import { LiveConnect } from 'live-connect-js'; // eslint-disable prebid/validate-imports
```

Comment:
This eslint rule currently fails for this import, but the new version developed [here](https://github.com/prebid/Prebid.js/pull/8293) allows it.